src/Tasks/dependency-check-build-task/dependency-check-build-task.ts

```diff
@@ -212,8 +285,31 @@ async function getZipUrl(version: string): Promise<void> {
 async function unzipFromUrl(zipUrl: string, unzipLocation: string): Promise<void> {
     let fileName = path.basename(url.parse(zipUrl).pathname);
     let zipLocation = tl.resolve(fileName)
+    let tmpError = null;
+    let response = null;
+    let downloadErrorRetries = 5;
+    
+    do {
```

Comment:
Retry downloading the dependency check ZIP up to 5 times (defined in variable downloadErrorRetries), needed for shakey network connections (we nearly always need to retry one time for example).